### PR TITLE
Update README: add SHA-1 checking example, change copyright year

### PR DESCRIPTION
**Pull request by ajlanghorn**
Updates the README to:
- add an example of how to check a SHA-1 of a file
- change the copyright year in the LICENSE paragraph

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ A Puppet module to download files with curl, supporting authentication.
 	  destination => "/tmp/index.html",
 	  timeout     => 0,
 	  verbose     => false,
+    sha         => "F79B40AEBC81B822F6096D606098CDA62C1997CD",
```
Comment (ajlanghorn) [inline README.md]: Weird; the spacing is fine locally...

File: README.md [hunk 2]
```diff
@@ -22,7 +23,7 @@ A Puppet module to download files with curl, supporting authentication.
 
 # License
 
-Copyright (c) 2013 Government Digital Services
+Copyright (c) 2015 Government Digital Service
```
Comment (mattbostock) [inline README.md]: The copyright year should be the year that the software was first release, so it doesn't need updating.

:+1: to the typo fix though.

File: README.md (hunk 1 above)
Comment (mattbostock) [inline README.md]: Hard tabs versus spaces?

Comment (ajlanghorn) [inline README.md]: Perhaps.

**Comment by ajlanghorn** [comment]
Closing pending more testing in Vagrant.

**Pull request closed without merging**